cli/src/lib.rs

```diff
@@ -337,8 +277,13 @@ impl Iroha {
             kura,
             block_sync,
             torii,
-            thread_handlers: vec![sumeragi_thread_handler, kura_thread_handler],
-            _sumeragi_relay: sumeragi_relay,
+            thread_handlers: vec![
+                sumeragi_thread_handler,
+                block_sync_thread_handler,
+                kura_thread_handler,
+                p2p_thread_handler1,
+                p2p_thread_handler2,
```

Comment:
We have thing number 1 and thing number 2. Are there really no distinguishing features between the two?